program/tests/test_wrap.rs

```diff
@@ -83,10 +85,16 @@ fn assert_wrap_result(starting_amount: u64, wrap_amount: u64, wrap_result: &Wrap
     );
     assert_eq!(recipient_token.mint, wrap_result.wrapped_mint.key);
 
-    // // Verify wrapped mint supply increased
-    // let wrapped_mint =
-    //     spl_token_2022::state::Mint::unpack(&wrap_result.wrapped_mint.
-    // account.data).unwrap(); assert_eq!(wrapped_mint.supply, wrap_amount);
+    // Verify wrapped mint supply increased
+    if wrap_result.wrapped_mint.account.owner == spl_token::id() {
+        let mint = spl_token::state::Mint::unpack(&wrap_result.wrapped_mint.account.data).unwrap();
+        assert_eq!(mint.supply, MINT_SUPPLY + wrap_amount);
+    } else {
+        let mint =
+            PodStateWithExtensions::<PodMint>::unpack(&wrap_result.wrapped_mint.account.data)
+                .unwrap();
```

Comment:
This works for any mint on token or token-2022, so feel free to use it without checking the owner!